### PR TITLE
CNN model for plate text recognition

### PR DESCRIPTION
**Pull request by jedyeo**
WIP PR. Separate repo for training our plate model which we'll access from inside the controller class.

### CODE REVIEW

**Comment by jedyeo** [comment]
https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit_generator
https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/image/ImageDataGenerator
https://www.pyimagesearch.com/2018/12/24/how-to-use-keras-fit-and-fit_generator-a-hands-on-tutorial/

for reference on datagen

**Comment by jedyeo** [comment]
`fit(
    x=None, y=None, batch_size=None, epochs=1, verbose=1, callbacks=None,
    validation_split=0.0, validation_data=None, shuffle=True, class_weight=None,
    sample_weight=None, initial_epoch=0, steps_per_epoch=None,
    validation_steps=None, validation_batch_size=None, validation_freq=1,
    max_queue_size=10, workers=1, use_multiprocessing=False
)`

**Comment by jedyeo** [comment]
`fit_generator(
    generator, steps_per_epoch=None, epochs=1, verbose=1, callbacks=None,
    validation_data=None, validation_steps=None, validation_freq=1,
    class_weight=None, max_queue_size=10, workers=1, use_multiprocessing=False,
    shuffle=True, initial_epoch=0
)`

**Comment by jedyeo** [comment]
Epoch bar when augmenting: why is it 40/40?

Epoch 1/3
40/40 [==============================] - 40s 1s/step - loss: 2.9579 - acc: 0.3344 - val_loss: 1.4889 - val_acc: 0.6719

Epoch bar when augmenting: out of 1280?
Epoch 1/5
1280/1280 [==============================] - 43s 34ms/sample - loss: 0.0080 - acc: 1.0000 - val_loss: 0.0010 - val_acc: 1.0000
